charts/gatekeeper-operator/templates/post-job.yaml

```diff
@@ -43,5 +43,6 @@ spec:
               memory: 16Mi
           securityContext:
             runAsNonRoot: true
+            runAsUser: 65534
```

Comment:
why is this needed? we chose to use `runAsNonRoot` which provides the same security as choosing a non-root UID. I recommend to close this PR

Reply:
we should remove anything related to real `PodSecurityPolicy` resources asap.

Reply:
I also want to remove the ability to be able to set the UID globally via policy at all, as it serves no use case

Reply:
well, PSP are in 3rd party charts

Reply:
> we should remove anything related to real `PodSecurityPolicy` resources asap.

I do not consider it as a "hot issue", please create a user story and put it to the backlog

Reply:
I just read your bug report and did not know this image was using root. (Why would it?)

So yes, if the uid can be set then great